packages/vite/src/node/shortcuts.ts

```diff
@@ -50,14 +50,16 @@ export function bindShortcuts(
 
     if (input === 'h') {
       server.config.logger.info(
-        shortcuts
-          .map(
+        [
+          '',
+          colors.bold('  Watch Usage'),
```

Comment:
Maybe `Shortcuts` or `Commands`? `Watch Usage` makes sense in the context of `vitest watch`.
```suggestion
          colors.bold('  Shortcuts'),
```